src/content/docs/fr/2/guide/notification.mdx

```diff
@@ -0,0 +1,161 @@
+
+    {/* TODO: Revise once https://github.com/tauri-apps/tauri/issues/7696 is in */}
+
+    ```rust
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialisation du plugin
+            .plugin(tauri_plugin_notification::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. Executez `cargo add tauri-plugin-notification` pour ajouter le plugin aux dépendances du projet dans `Cargo.toml`.
+
+    2. Modifiez `lib.rs` Pour initialisez le plugin:
+
+    ```rust
+    // lib.rs
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialisation du plugin
+            .plugin(tauri_plugin_notification::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    3. Si vous voulez utiliser les notification de Javascript, vous devez aussi installer le package npm: If you'd like to use notifications in JavaScript then install the npm package as well:
+
+    <CommandTabs
+        npm="npm install @tauri-apps/plugin-notification"
+        yarn="yarn add @tauri-apps/plugin-notification"
+        pnpm="pnpm add @tauri-apps/plugin-notification"
+    />
+
+    </TabItem>
+
+</Tabs>
+
+## Utilisation :
+
+Voici quelques exemples montrant comment utiliser le plugin de notification:
+
+- [Envoyer une notification aux utilisateurs](#envoyer-une-notification)
+- [Ajouter une action à une notification](#actions)
+- [Ajouter une pièce jointe à une notification](#pièce-jointe)
+- [Envoyer une notification dans un channel spécifique](#channels)
+
+{/* TODO: Link to which language to use, frontend vs. backend guide when it's made */}
+
+Le plugin de notification est disponible aussi bien en Javascript qu'en Rust.
+
+### Envoyer une Notification
+
+{/* TODO: Demo component */}
+
+Suivez ces étapes pour envoyer une notification:
+
+1. Vérifier si la permission est accordée
+2. Demander la permission si elle n’est pas accordée
+3. Envoyer la notification
+
+<Tabs>
+<TabItem label="JavaScript">
+
+```js
+import {
+	isPermissionGranted,
+	requestPermission,
+	sendNotification,
+} from '@tauri-apps/plugin-notification';
+
+// Avez-vous la permission d'envoyer une notification ?
+let permissionGranted = await isPermissionGranted();
+
+// Le cas échéant on la demande
+if (!permissionGranted) {
+	const permission = await requestPermission();
+	permissionGranted = permission === 'granted';
+}
+
+// Une fois la permission obtenue, on envoie la notification
+if (permissionGranted) {
+	sendNotification({ title: 'Tauri', body: 'Tauri is awesome!' });
+}
+```
+
+</TabItem>
+<TabItem label="Rust">
+
+{/* TODO: */}
+
+<Stub />
+
+</TabItem>
+</Tabs>
+
+### Actions
+
+{/* TODO: */}
+
+<Stub />
+
+### Pièce-jointe

```

Comment:
Since this will refer to a specific API usage for notifications, I think it makes sense to keep this in English? Then when the copy is added below this heading the French term "Pièce-jointe" could be used there?

Reply:
Yeah for now it isn't a problem, fixed